### ISSUE TITLE
Wrong when running grid.py with python3.6

### ISSUE THREAD

**Issue by pinckCat** [pyecharts/pyecharts-snapshot#38]
File "grid.py", line 4, in <module>
    from pyecharts import Bar, Grid, Line, Scatter, EffectScatter
ImportError: cannot import name 'Bar'

**Comment by chfw** [comment]
Yep,  you are right. The example is outdated. Please use older version of pyecharts 0.5.x and pyecharts-snapshot.

I will have to delete these examples.

With the latest pyecharts-snapshot, only cli command is supported and will not work with inline code. For similar inline usage, please read pyecharts v1.0.0 docs. Yes it is still possible but this library do not support it. New style libraries are snapshot- selenium and snapshot-phantomjs.

Good issue.
